src/plotApplications/components/PlotApplication.js

```diff
@@ -135,6 +138,10 @@ const SingleSectionItem = ({ section, answer, fieldTypes, plotSearch, topLevel,
 };
 
 const SectionData = ({section, answer, topLevel = false, fieldTypes, plotSearch }) => {
+  if (!answer) {
```

Comment:
This 'answer' Variable is a bit confusing to me.

It is sometimes handled as an object (fe. SingleSectionItem -component there is `const fieldAnswer = answer.fields[field.identifier];`

Where as in row 187 it is passed as array :thinking:

Reply:
`answer` can both be an object (with `fields`, `sections` and possibly `metadata`) or an array of such objects. It's the latter only if the section it relates to can be duplicated automatically (i.e. top level target sections) or manually (i.e. top level applicant sections, or any other ones set in the form as such). Line 158 makes this distinction by reading `section.add_new_allowed`, and the handling inside the ternary branches should be consistent with that.

As long as the form itself hasn't been modified after the application was sent in, which should be the case given we now lock that when the application period has started, these should always agree with the form in the application data.